tests/features/spicy-protocol-analyzer/tcp-one-unit

```diff
@@ -0,0 +1,5 @@
+# @TEST-DOC: Smoke test for TCP spicy-protocol-analyzer with one unit.
+
+# @TEST-EXEC: ${SCRIPTS}/zkg -vvvvv create --packagedir test --features=spicy-protocol-analyzer --user-var name=MyPackage --user-var analyzer=MyAnalyzer --user-var protocol=Tcp --user-var unit_orig=MyMessage --user-var unit_resp=MyMessage
```

Comment:
Here and elsewhere: this drops the `${rnd}`. I think we should keep that, it saved me a couple of times during development.